zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -776,7 +776,7 @@ autoupdate.api.view.updatedAddons = Return a list of any add-ons that have been
 autoupdate.desc = Allows ZAP to check for updates
 autoupdate.name = Auto-update Extension
 
-break.api.action.addHttpBreakpoint = Adds a custom HTTP breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.  
+break.api.action.addHttpBreakpoint = Adds a Custom HTTP Breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.  
```

Comment:
This one could be left since its a sentence with a period. (It is used in a web UI component not in ZAP's GUI.)

Reply:
Right, makes sense, this change has been reversed.